charts/jx-app-jacoco/Makefile

```diff
@@ -38,10 +38,10 @@ else ifeq ($(OS),Linux)
 	sed -i -e "s/version:.*/version: $(VERSION)/" Chart.yaml
 	sed -i -e "s/tag: .*/tag: $(VERSION)/" values.yaml
 else
-	echo "platfrom $(OS) not supported to tag with"
+	echo "platform $(OS) not supported to tag with"
 	exit -1
 endif
 	git add --all
-	git commit -m "release $(VERSION)" --allow-empty # if first release then no verion update is performed
+	git commit -m "release $(VERSION)" --allow-empty # if first release then no version update is performed
 	git tag -fa v$(VERSION) -m "Release version $(VERSION)"
-	git push origin v$(VERSION)
\ No newline at end of file
+	git push origin HEAD v$(VERSION)
```

Comment:
I think there are two options:

* Skip the commit and just tag. Effectively the HEAD of master gets tagged for the release. This will work I guess and one does not have to deal with endless loops etc. However, it also means that when checking out the tag once cannot directly re-build the artifacts. One needs to create the VERSION file first
* The second option is really that one keeps VERSION under version control. It gets updated when a new release build occurs (Jenkinsfile). Calling this tag target will also update the version in Chart.yaml and values.yaml (remember another confusion we discussed before). I think this was the original intent here and some process I would recognize. For that to happen we need to stop the triggering of yet another release build when we push version commit.

Reply:
The second option seems like the best approach: Try adding `run_if_changed: ^(?!VERSION).*$` to the `postsubmit` config for the job. You can't avoid triggering on new commits, but you can avoid triggering on particular files of these two.